base/services/nfsd/readdir.c

```diff
@@ -165,7 +165,11 @@ static void readdir_copy_shortname(
     /* GetShortPathName returns number of characters, not including \0 */
     *name_size_out = (CCHAR)GetShortPathNameW(name, name_out, 12);
     if (*name_size_out) {
+#ifndef __REACTOS__
```

Comment:
Is this 3rd party code? Care to upstream the fix?

Reply:
Yes. Not now.
NB:
First, that import needs to be added to `media/doc/README.FSD`: upstream source, forked or sync'ed, ...